src/main/java/qupath/ext/instanseg/core/MatTranslator.java

```diff
@@ -23,10 +31,11 @@ class MatTranslator implements Translator<Mat, Mat> {
      * @param outputChannels Array of channels to output; if null or empty, output all channels.
      *                       Values should be true for channels to output, false for channels to ignore.
      */
-    MatTranslator(String inputLayoutNd, String outputLayoutNd, boolean[] outputChannels) {
+    MatTranslator(String inputLayoutNd, String outputLayoutNd, boolean[] outputChannels, Map<String, ?> optionalArgs) {
         this.inputLayoutNd = inputLayoutNd;
         this.outputLayoutNd = outputLayoutNd;
         this.outputChannels = convertBooleanArray(outputChannels);
+        this.optionalArgs = optionalArgs;
```

Comment:
I know it's (currently) not in the public API, but I feel safer making a defensive copy...

Reply:
I thought the one defensive copy was enough, but fair